src/main/index.js

```diff
@@ -29,6 +29,8 @@ let mainWindow;
 
 async function createMainWindow() {
   const window = new BrowserWindow({
+    width: 1200,
+    height: 900,
```

Comment:
Could this be moved to a separate commit, or dropped from the PR? Window size seems irrelevant for clipboard stuff.

Reply:
Ha! This actually wasn't even meant to go into this one, think it slipped in when I had a `git commit --amend --signoff` I'll pull it back. I was going to see about this though as I think the current default window size is pretty tiny, I always have to make it larger every time I open Chrysalis.

Reply:
It might be worth saving the window size, position, etc and restore that on startup? But lets discuss that in a real ticket on its own :)

Reply:
Will open this up now.

Reply:
Reverted that change